api/db/populate-friendships.js

```diff
@@ -0,0 +1,85 @@
+const { getRandomSubset, getRandomDate, createPrompt } = require('./utils');
+
+const User = require('../models/User');
+const Friendship = require('../models/Friendship');
+
+let displayLogs;
+
+const createFriendship = async (friendshipParams) => {
+  const friendship = new Friendship({
+    requestant: friendshipParams.requestant,
+    receivant: friendshipParams.receivant,
+    accepted: friendshipParams.accepted,
+    acceptedAt: friendshipParams.acceptedAt,
+    lastMessageAt: friendshipParams.lastMessageAt,
```

Comment:
I think we should set this when populating the messages collection instead, but maybe I'm not seeing your intention here.

Reply:
Do you mean instead of having it be it's own script command?

Reply:
Let me try to explain myself better:

When a friendship is created or accepted, the `lastMessageAt` value doesn't get set, it only gets set when a message between both users is actually sent. Thus, I think we should handle setting this value when populating the messages collection.

Reply:
Gotcha, that makes sense. Are you opposed to keeping it as is until I work on the messages collection script?

Reply:
I think I would remove it, as this as is can be misleading for developers visiting the `/chats` route, although seeing how quickly you are adding all scripts this won't be a problem for too long 😛

Reply:
After giving this a second thought, maybe it will be better to remove it to avoid finding unnecessary constraints when populating the messages collection?